docs/developer-guide/modules-development-via-docker.md

```diff
@@ -1,71 +1,130 @@
+
+For the *docker* and *docker-compose* files configuring convenience, all configurable parameters have placed on separate *.env* file. In the *.env* file you can set values for:
+
+* **PLATFORM_VERSION** - tag for required *virtocommerce/platform* image (default value `dev-linux-latest`)
+* **MODULES_VOLUME** - path to the `./Modules` folder on host machine
+* **APP_DATA_MODULES** - path to the `./App_Data/Modules` folder on host machine
+* **DB_PASS** - SQl Server service password
+* **REDIS_PASS** - Redis service password (default value `passwd@123`)
+* **SEARCH_PROVIDER** - used in VC Platform search provider (default value ElasticSearch)
+
+## Prerequisites
+
+* You need to have some basic understanding of [Docker](https://docs.docker.com/get-started/), [Docker Compose](https://docs.docker.com/compose/gettingstarted/), and the key terms used in the ecosystem
+* [Docker Desktop for Windows](https://docs.docker.com/docker-for-windows/install/) installed on your machine
+* Create GitHub [personal access token](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token)
+* [Authenticate to GitHub Packages](https://docs.github.com/en/packages/using-github-packages-with-your-projects-ecosystem/configuring-docker-for-use-with-github-packages#authenticating-to-github-packages)
+
+## How to install Docker for Windows
+
+* For Docker installation, first review the information at Docker for Windows: [What to know before you install](https://docs.docker.com/docker-for-windows/install/#what-to-know-before-you-install)
+* Install [Docker Desktop for Windows](https://docs.docker.com/docker-for-windows/install/) on your machine
+* During installation you'll need to choose Linux as operating system used inside your containers
 
-> You can parameterize these values ​​in the .env file
+## How to use
 
+1. Copy [ModulesDevelop](https://github.com/VirtoCommerce/vc-platform/blob/dev/DockerCompose/ModulesDevelop/) folder to local machine
+2. Create an external network for the Docker engine
+
+```cmd
+docker network create nat
 ```
+
+3. Parameterize values in the *.env* file.
+
+```cmd
 CMS_CONTENT_VOLUME=/Path/to/folder/cms-content
```

Comment:
we're talking about windows environment, so lets use windows path instead of linux one:
```suggestion
CMS_CONTENT_VOLUME=c:\path\to\folder\cms-content
```